assets/javascripts/discourse/components/explorer-schema.hbs

```diff
@@ -4,13 +4,11 @@
     @icon="chevron-left"
     @class="no-text unhide"
   />
-{{/if}}
-
-<div class="schema">
-  <div class={{if this.hideSchema "hidden"}}>
+{{else}}
+  <div class="schema">
     <div class="schema-search inline-form full-width">
       <TextField
-        @value={{this.filter}}
+        @value={{this.filterInput}}
```

Comment:
If `filterInput` isn't read anywhere, and its value is set only by this textfield then I think it's not needed at all?

Reply:
Good catch. Our `TextField` wrapper needed it, but `Input` doesn't need it, and using `Input` is cleaner (fewer debounces as well).